scripts/build_podspec.sh

```diff
@@ -74,7 +74,7 @@ Pod::Spec.new do |s|
   s.documentation_url = 'https://apple.github.io/swift-log'
```

Comment:
We should delete this file

Reply:
Agreed, kill it

Reply:
Yeah okay I wasn't sure if we were still supporting this or not. FYI: it's still in swift-metrics, so we may want to get rid of it too.